Author: Denis Kristak (Inuits)
Removing packages that are already in standard Python library, also using manual versioning instead of setuptools_scm.
diff -ruN scenicplus-1.0.0_orig/requirements.txt scenicplus-1.0.0/requirements.txt
--- scenicplus-1.0.0_orig/requirements.txt	2023-03-13 08:28:38.000000000 +0000
+++ scenicplus-1.0.0/requirements.txt	2023-09-26 10:26:49.423833827 +0100
@@ -3,10 +3,8 @@
 tqdm
 ray
 attr
-typing
 matplotlib
 scikit-learn
-statistics
 pyranges
 scipy
 pybiomart
diff -ruN scenicplus-1.0.0_orig/setup.py scenicplus-1.0.0/setup.py
--- scenicplus-1.0.0_orig/setup.py	2023-03-13 08:28:38.000000000 +0000
+++ scenicplus-1.0.0/setup.py	2023-09-25 14:02:23.271569830 +0100
@@ -30,7 +30,7 @@
 
 setuptools.setup(
      name='scenicplus',
-     use_scm_version=True,
+     version='1.0.0',
      setup_requires=['setuptools_scm'],
      packages=setuptools.find_packages(where='src'),
      package_dir={'': 'src'},
